src/main/java/seedu/address/logic/parser/FindPersonCommandParser.java

```diff
@@ -152,7 +153,7 @@ private void checkModuleValidity(String[] modules) throws ParseException {
 
         pattern = Pattern.compile(VALIDATION_REGEX);
         for (String s : modules) {
-            matcher = pattern.matcher(s);
+            matcher = pattern.matcher(s.toUpperCase(Locale.ROOT));
```

Comment:
I am not sure if this line is necessary as the current iteration of find is able to match module code in lower case with module code in uppercase.

Reply:
This is required to match the VALIDATION_REGEX as this allows the lowercase to be forced to uppercase to match it. This introduces the case insensitivity.